src/js/stores/AppsStore.js

```diff
@@ -10,6 +11,24 @@ function removeApp(apps, appId) {
   }).value();
 }
 
+function removeTask(tasks, relatedAppId, taskId) {
+  return lazy(tasks).reject(function (task) {
+    return task.id === taskId && task.appId === relatedAppId;
+  }).value();
+}
+
+function processCurrentApp(currentApp) {
```

Comment:
Hmm, I am not sure, because we do a 'processing' of already existing json-data, nothing to initialize here.
And I think it could be performance problem to convert a simple object (that is only gets some lazy.js transformations on some point) into a class with an constructor, if thinking at scale.
But in generally I would love to introduce some ES6! :)